pkg/cmd/admin.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/openshift/microshift/pkg/admin/data"
+	"github.com/openshift/microshift/pkg/config"
+	"github.com/openshift/microshift/pkg/version"
+
+	"github.com/spf13/cobra"
+)
+
+func newAdminDataCommand() *cobra.Command {
+	backup := &cobra.Command{
+		Use:   "backup",
+		Short: "Backup MicroShift data",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return data.NewManager().Backup(
+				data.BackupConfig{
+					Storage: cmd.Flag("storage").Value.String(),
+					Name:    cmd.Flag("name").Value.String(),
+				},
+			)
+		},
+	}
+
+	data := &cobra.Command{
+		Use:   "data",
+		Short: "Commands for managing MicroShift data",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if cmd.Flag("storage").Value.String() == "" {
+				return fmt.Errorf("--storage must not be empty")
+			}
+
+			if cmd.Flag("name").Value.String() == "" {
+				return fmt.Errorf("--name must not be empty")
+			}
+
+			if err := data.MicroShiftIsNotRunning(); err != nil {
+				return fmt.Errorf("microshift must not be running: %w", err)
+			}
+
+			return nil
+		},
+	}
+	v := version.Get()
+	data.PersistentFlags().String(
```

Comment:
What's the use case for making the storage location configurable from the command line?

Reply:
I'd say nice to have for RPM systems where backups are manual. But i don't have concrete use case other than NFS (or different partition) mount for backups, but then CoW goes out of window

Reply:
OK, that makes sense. In that case I think we're going to not want to overwrite an existing backup, so we should think about how to handle that. Maybe just a check in the CLI code, before we get into the low-level code, where we do want old backups overwritten.

Reply:
Added `BackupExists()` and `--force`